Sources/FluidAudio/TextToSpeech/Kokoro/Voices/VoiceEmbeddingDownloader.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+/// Downloads voice embeddings from HuggingFace
+public enum VoiceEmbeddingDownloader {
```

Comment:
seems like the extension for voices in Download Utils could be moved here as well